office/ProjectLibre/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           java 1.0
+
+# retrieving shasum 858bc0d6f82fdea6c67f145c768e70b48beb1b16 from master because there is no tag 1.8.0 set
+name                ProjectLibre
+version             1.8.0-git
+
+java.version        1.8
+java.fallback       openjdk8
+
+revision            0
+categories          office java
+platforms           darwin
+maintainers         {gmail.com:giansalvo.gusinu @giansalvo} openmaintainer
+license             GPL-2
+
+description         ProjectLibre is a project management software.
+
+long_description \
+    ProjectLibre is an open source desktop alternative to Microsoft Project. \
+    It has been downloaded over 3,000,000 times in over 200 countries and has \
+    won InfoWorld "Best of Open Source" award. If you are looking for a team \
+    solution or multi-project management we recommend our cloud version. \
+    ProjectLibre has a lengthy list of Fortune 500, Government, small business \
+    and non-profits utilizing our comprehensive project management features. \
+    ProjectLibre has been translated into 20 languages.
+
+master_sites        sourceforge:code-snapshots/git/p/pr/projectlibre/code.git
+distfiles           projectlibre-code-858bc0d6f82fdea6c67f145c768e70b48beb1b16.zip
+worksrcdir          projectlibre-code-858bc0d6f82fdea6c67f145c768e70b48beb1b16
```

Comment:
If you fetch from release files, these are not needed.

Reply:
see previous comment